test/inductor/test_cudagraph_trees.py

```diff
@@ -1851,6 +1851,18 @@ def foo(x):
             out = foo(torch.rand([4, 4], device="cuda", requires_grad=True))
             self.assertFalse(self.get_manager().new_graph_id().id == 0)
 
+        def test_input_unbacked_symint(self):
+            @torch.compile(mode="reduce-overhead")
+            def foo(x):
+                return x + 1
+
+            x = torch.randn(2, 3, device="cuda")
+            torch._dynamo.decorators.mark_unbacked(x, 0)
+
+            foo(x)
+
+            self.assertEqual(counters["inductor"]["cudagraph_skips"], 0)
```

Comment:
does this mean we now fully support `mark_unbacked` with cudagraphs? Or are there cases you can think of (that we should probably document somewhere) where we can expect cudagraphs not to work.

One I can imagine is whether any of the unbacked symints generate runtime asserts. Which I imagine will not naively be cudagraph-able?